packages/common/src/core/schemas/internal/metrics/StatCardUiSchema.ts

```diff
@@ -344,19 +362,11 @@ const HIDE_FOR_ALL = {
   },
 };
 
-const HIDE_FOR_DATA_VIZ_RULE = {
-  effect: UiSchemaRuleEffects.HIDE,
-  condition: {
-    scope: "/properties/layout",
-    schema: { enum: ["dataViz"] },
-  },
-};
-
 const SHOW_FOR_MORE_INFO_RULE = {
```

Comment:
Let's update the name of this rule, now that it's called "Informational" and not "More info"